substrate-node/node/src/service.rs

```diff
@@ -88,6 +89,18 @@ pub fn new_partial(
 		)?;
 	let client = Arc::new(client);
 
+	let keystore = keystore_container.sync_keystore();
+	if config.offchain_worker.enabled {
+		// Initialize seed for signing transaction using offchain workers. This is a convenience
+		// so learners can see the transactions submitted simply running the node.
+		// Typically these keys should be inserted with RPC calls to `author_insertKey`.
+		sp_keystore::SyncCryptoStore::sr25519_generate_new(
+			&*keystore,
+			tfchain_runtime::pallet_tft_price::KEY_TYPE,
+			Some("//Alice"),
+		).expect("Creating key with account Alice should succeed.");
+	}
+
```

Comment:
Why is this here?

Reply:
So the price tft price pallet can be tested in a dev environment without having to add a key manually

Reply:
removed it to be added later